packages/cypress-commands/src/commands/login.js

```diff
@@ -53,5 +53,14 @@ const loginDev = () => {
  * https://docs.cypress.io/guides/guides/web-security.html#One-Superdomain-per-Test
```

Comment:
I think this comment has become misplaced, it probably should be above `loginBasicAuth`?